pwndbg/commands/telescope.py

```diff
@@ -39,24 +40,27 @@ def telescope(address=None, count=telescope_lines, to_string=False):
     ($sp by default)
     """
     address = int(address if address else pwndbg.regs.sp) & pwndbg.arch.ptrmask
-    count   = int(count) & pwndbg.arch.ptrmask
+    count   = max(int(count), 0) & pwndbg.arch.ptrmask
```

Comment:
sure, sounds fair. will make it print something and return... but will need that check also for address a bit later as one could pass `telescope $sp $sp` :smile: